community-version.py

```diff
@@ -397,3 +397,35 @@ def get_image_path():
                                 brightness=float(args.brightness) if args.brightness else 1.0,
                                 output_image=args.output_imagek
                                 )
+
+
+def convert_image_to_ascii(image, new_width=100):
+    """
+    Returns ascii art in coloured form in combination with `to_color()` function down below
+    """
+    image = scale_image(image)
+    image = convert_to_grayscale(image)
+
+    pixels_to_chars = map_pixels_to_ascii_chars(image)
+    len_pixels_to_chars = len(pixels_to_chars)
+
+    image_ascii = [pixels_to_chars[index: index + new_width] for index in
+                   range(0, len_pixels_to_chars, new_width)]
+
+    # return "\n".join(image_ascii)
+    # Output is given to `to_color()` function which makes the text coloured
+    return to_color("\n".join(image_ascii), 'blue')
```

Comment:
since this is the only change, perhaps you can apply this color filter in a better place. Possibly before the output is rendered. It should be conditional too.